### ISSUE TITLE
`nameko backpoort` asking options, which are not used

### ISSUE THREAD

**Issue by vlcinsky** [nameko/nameko#623]
```bash
$ nameko backdoor --help
usage: nameko backdoor [-h] [--rlwrap] [-c CONFIG] [-d KEY=VALUE] [host:]port

Connect to a nameko backdoor. If a backdoor is running this will connect to a
remote shell. The runner is generally available as `runner`.

positional arguments:
  [host:]port           (host and) port to connect to

optional arguments:
  -h, --help            show this help message and exit
  --rlwrap, --no-rlwrap
                        Use rlwrap
  -c CONFIG, --config CONFIG
                        The YAML configuration file
  -d KEY=VALUE, --define KEY=VALUE
                        Set config entry. Overrides value loaded from config
                        file. Can be used multiple times. Example: --define
                        AMQP_URI=pyamqp://guest:guest@localhost
```

Helpstring explains use of `--config` and `--define` options, but they are not used in real code at all.

Expected behaviour: Remove usage of `--config` and `--define` options from CLI.

I just want to make sure, this change is correct and acceptable for nameko 3.x. I already work on it in  my PR #620  for refactored CLI.

**Comment by mattbennett** [comment]
Completely correct. These arguments are not valid for the `backdoor` command.

But superseded by the discussion in https://github.com/nameko/nameko/issues/625 :)

**Comment by vlcinsky** [comment]
Closing as it will be resolved by #625 (removal of `nameko backdoor` subcommand)
